pkg/metering/units.go

```diff
@@ -0,0 +1,25 @@
+package metering
+
+const (
+	payloadName = "payload"
+	payloadUnit = "bytes"
+
+	computeName = "compute"
+	computeUnit = "seconds"
+)
+
+type MeteringUnit struct {
```

Comment:
The linter should be questioning the redundant name: `metering.MeteringUnit`. How about just `metering.Unit`?
```suggestion
type Unit struct {
```

Reply:
3edcee8188f82fb8907ce8e4eaaafc46e6875548